sweep/sweeper.go

```diff
@@ -928,7 +928,6 @@ func (s *UtxoSweeper) clusterByLockTime(inputs pendingInputs) ([]inputCluster,
 	pendingInputs) {
 
 	locktimes := make(map[uint32]pendingInputs)
-	inputFeeRates := make(map[wire.OutPoint]chainfee.SatPerKWeight)
```

Comment:
Solid change, maybe we added the fee rate attribute later on?

Reply:
did a git blame and it seems it was added before